charts/latest/azurefile-csi-driver/templates/csi-azurefile-controller.yaml

```diff
@@ -22,6 +22,7 @@ spec:
         - name: csi-provisioner
           image: "{{ .Values.image.csiProvisioner.repository }}:{{ .Values.image.csiProvisioner.tag }}"
           args:
+            - "-v=5"
```

Comment:
5 seems too verbose, how about using 3?

Reply:
I would suggest as 5 to provide more detailed logs, `csi-provisioner` only do the provision work, it won't have too many logs.